synphot/models.py

```diff
@@ -587,6 +618,16 @@ def sampleset(self, factor_step=0.1, **kwargs):
 
         return np.asarray(w)
 
+    # TODO: Remove unit hardcoding when we use model with units natively.
+    def integrate(self, x):
+        # TODO: Fix the formula, apply units, check for n_models>1
```

Comment:
Something is not quite right with this one. @eslavich , since you suggested the first derivative of Gaussian for this, maybe you can check my rusty calculus? Or @mcara ?

Reply:
Corrected equation:
```python
return self.amplitude * (a2 - a1)
```

Reply:
Thank you!